go-backend/db/user.go

```diff
@@ -6,17 +6,127 @@ import (
 	logger "github.com/sirupsen/logrus"
 )
 
+const (
+	getUserQuery = `SELECT id,
+		org_id,
+		first_name,
+		last_name,
+		email,
+		display_name,
+		profile_image,
+		soft_delete,
+		role_id,
+		hi5_quota_balance,
+		soft_delete_by,
+		soft_delete_at
+		FROM users WHERE id=$1`
+
+	updateUserQuery = `UPDATE users SET (
+		first_name,
+		last_name,
+		email,
+		display_name,
+		profile_image,
+		soft_delete,
+		role_id,
+		hi5_quota_balance,
+		soft_delete_by,
```

Comment:
I don't think we should allow soft delete columns to be set via update API, there will be a separate delete API later which will set these fields